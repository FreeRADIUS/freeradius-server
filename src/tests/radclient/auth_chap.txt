#
#	ARGV: -s
#
User-Name = "chap-test",
Chap-Password = "topsecret"
CHAP-Challenge = 0x000102030405060708090a0b0c0d0e0f10111213
