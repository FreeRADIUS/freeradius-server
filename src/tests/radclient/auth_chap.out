Packet summary:
	Accepted      : 1
	Rejected      : 0
	Lost          : 0
	Errored       : 0
	Passed filter : 1
	Failed filter : 0
